guides/common/modules/proc_configuring-tracer-on-a-host.adoc

```diff
@@ -1,18 +1,25 @@
-[id="enabling-tracer-on-a-host_{context}"]
-= Enabling Tracer on a host
+[id="configuring-tracer-on-a-host_{context}"]
+= Configuring Tracer on a host
 
-Use this procedure to enable Tracer on {Project} and access Traces.
-Tracer displays a list of services and applications that need to be restarted.
+You can install Tracer on hosts and access Traces on {Project}.
+Tracer displays a list of services and applications that are outdated and need to be restarted.
 Traces is the output generated by Tracer in the {ProjectWebUI}.
 
 .Prerequisites
+* The host is registered to {ProjectName}.
 include::snip_prerequisite-project-client-repository-enabled.adoc[]
 * Remote execution is enabled.
 
 .Procedure
+
 . In the {ProjectWebUI}, navigate to *Hosts* > *All Hosts*.
 . Click the name of the host you want to modify.
 . On the *Traces* tab, click *Enable Traces*.
 . Select the provider to install `katello-host-tools-tracer` from the list.
 . Click *Enable Tracer*.
 You get a REX job notification after the remote execution job is complete.
+. Upload the Tracer data to your {ProjectServer}:
```

Comment:
```suggestion
. On your host, upload the Tracer data to {Project}:
```

To me, it's obvious that you do this over SSH or via REX. I am ensure if we should add the following:

```suggestion
. Start a remote execution job to upload the Tracer data from your host to {Project}:
+
For more information, see xref:Configuring_and_Setting_Up_Remote_Jobs_{context}[].
```